src/Database/Type/DateTimeType.php

```diff
@@ -352,7 +352,8 @@ protected function _setClassName(string $class, string $fallback): void
             $class = $fallback;
         }
         $this->_className = $class;
-        $this->_datetimeInstance = new $this->_className;
+        // @codingStandardsIgnoreLine
```

Comment:
These ignores can be removed once stickler is updated to use phpcs `3.3.1`.

Reply:
That's done now :+1: